pom.xml

```diff
@@ -0,0 +1,89 @@
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.eclipse.pass</groupId>
+  <artifactId>eclipse-pass-parent</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Eclipse-PASS Parent</name>
+  <url>https://github.com/eclipse-pass/main</url>
+
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>Jim Martino</name>
+      <email>jrm.jhu@jhu.edu</email>
+      <organization>The Sheridan Libraries, Johns Hopkins Univeristy</organization>
+      <organizationUrl>https://www.library.jhu.edu/</organizationUrl>
+    </developer>
+    <developer>
+      <name>Mark Patton</name>
+      <email>mpatton@jhu.edu</email>
+      <organization>The Sheridan Libraries, Johns Hopkins Univeristy</organization>
+      <organizationUrl>https://www.library.jhu.edu/</organizationUrl>
+    </developer>
+    <developer>
+      <name>John Abrahams</name>
+      <email>jabrah20@jhu.edu</email>
+      <organization>The Sheridan Libraries, Johns Hopkins Univeristy</organization>
+      <organizationUrl>https://www.library.jhu.edu/</organizationUrl>
+    </developer>
+  </developers>
+
+  <properties>
+    <duraspace-codestyle.version>1.1.0</duraspace-codestyle.version>
+  </properties>
+
+  <build>
+  <plugins>
+    <!-- Used to validate all code style rules in source code using Checkstyle -->
+    <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-checkstyle-plugin</artifactId>
+      <version>3.1.0</version>
+      <executions>
+        <execution>
+          <id>verify-style</id>
+          <!-- Bind to verify so it runs after package & unit tests, but before install -->
+          <phase>verify</phase>
+          <goals>
+            <goal>check</goal>
+          </goals>
+        </execution>
+      </executions>
+      <configuration>
+        <configLocation>duraspace-checkstyle/checkstyle.xml</configLocation>
+        <suppressionsLocation>duraspace-checkstyle/checkstyle-suppressions.xml</suppressionsLocation>
+        <encoding>UTF-8</encoding>
+        <consoleOutput>true</consoleOutput>
+        <logViolationsToConsole>true</logViolationsToConsole>
+        <failOnViolation>true</failOnViolation>
+        <includeTestSourceDirectory>true</includeTestSourceDirectory>
+      </configuration>
+      <dependencies>
+        <dependency>
+          <groupId>org.duraspace</groupId>
+          <artifactId>codestyle</artifactId>
+          <version>${duraspace-codestyle.version}</version>
+        </dependency>
+        <!-- Override dependencies to use latest version of checkstyle -->
+        <dependency>
+          <groupId>com.puppycrawl.tools</groupId>
+          <artifactId>checkstyle</artifactId>
+          <version>8.29</version>
```

Comment:
Could a more recent version of checkstyle be used?

Reply:
http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd
Suggests that we _should_ be able to run up to checkstyle v9.x, however, I was spot checking checkstyle plugin versions (manually changing the version in the IDEA plugin) and I'd get errors on any version above `8.41.1`. The error message suggested that the `checkstyle.xml` was invalid in versions `8.42+`

Error message in IDEA:
```
The Checkstyle rules file could not be parsed. cannot initialize module TreeWalker - cannot initialize module JavadocMethod - Property 'scope' does not exist, please check the documentation
```
Which looks like it refers to this part of the config: https://github.com/duraspace/codestyle/blob/master/src/main/resources/duraspace-checkstyle/checkstyle.xml#L98

Reply:
I've updated the checkstyle version to `8.41.1`, which also required an update to the maven-checkstyle-plugin to `3.1.2`